x-pack/plugins/spaces/server/usage_collection/spaces_usage_collector.ts

```diff
@@ -65,6 +67,12 @@ async function getSpacesUsage(
             size: knownFeatureIds.length,
           },
         },
+        solution: {
+          terms: {
+            field: 'space.solution',
+            size: knownSolutions.length,
+          },
+        },
```

Comment:
Will this handle spaces without a solution defined? You might be able to aggregate on those using the `missing` property of terms aggregations: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html#_missing_value_5

Reply:
Added spaces without a solution defined to `unset` bucket 👍